<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowTotal</class>
 <widget class="QMainWindow" name="MainWindowTotal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>职工考勤</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: #ffffff;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}
QMenuBar {
    background-color: #f5f5f5;
    color: #333333;
    font-size: 14px;
    border-bottom: 1px solid #dddddd;
}
QMenuBar::item {
    background-color: transparent;
    padding: 5px 10px;
}
QMenuBar::item:selected {
    background-color: #e0e0e0;
}
QMenu {
    background-color: #f5f5f5;
    color: #333333;
    border: 1px solid #dddddd;
}
QMenu::item:selected {
    background-color: #e0e0e0;
}
QToolBar {
    background-color: #f5f5f5;
    border: none;
    padding: 5px;
    border-bottom: 1px solid #dddddd;
}
QToolButton {
    background-color: transparent;
    color: #333333;
    padding: 5px;
    border-radius: 5px;
}
QToolButton:hover {
    background-color: #e0e0e0;
}
QTabWidget::pane {
    border: 1px solid #dddddd;
    background-color: #ffffff;
}
QTabBar::tab {
    background-color: #f5f5f5;
    color: #333333;
    padding: 10px;
    border: 1px solid #dddddd;
    border-bottom: none;
}
QTabBar::tab:selected {
    background-color: #ffffff;
    border-bottom: 1px solid #ffffff;
}
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #dddddd;
    gridline-color: #dddddd;
}
QHeaderView::section {
    background-color: #f5f5f5;
    color: #333333;
    padding: 5px;
    border: 1px solid #dddddd;
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>考勤记录</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionSignIn"/>
    <addaction name="actionSign_out"/>
    <addaction name="separator"/>
    <addaction name="actionLeave"/>
    <addaction name="actionPermit"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="separator"/>
    <addaction name="actionPerson"/>
    <addaction name="actionEmployeeInfo"/>
    <addaction name="separator"/>
    <addaction name="actionWorkTime"/>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSignIn"/>
   <addaction name="actionSign_out"/>
   <addaction name="separator"/>
   <addaction name="actionLeave"/>
   <addaction name="actionPermit"/>
   <addaction name="actionRecord"/>
   <addaction name="separator"/>
   <addaction name="actionPerson"/>
   <addaction name="actionEmployeeInfo"/>
   <addaction name="actionWorkTime"/>
   <addaction name="actionExport"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionSignIn">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/打卡.png</normaloff>:/images/打卡.png</iconset>
   </property>
   <property name="text">
    <string>签到</string>
   </property>
   <property name="toolTip">
    <string>签到</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionLeave">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/请假.png</normaloff>:/images/请假.png</iconset>
   </property>
   <property name="text">
    <string>请假</string>
   </property>
   <property name="toolTip">
    <string>请假</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPermit">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/事务.png</normaloff>:/images/事务.png</iconset>
   </property>
   <property name="text">
    <string>批假</string>
   </property>
   <property name="toolTip">
    <string>批假</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/icon/导出.png</normaloff>:/icon/导出.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>导出</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/台账.png</normaloff>:/images/台账.png</iconset>
   </property>
   <property name="text">
    <string>请假记录</string>
   </property>
   <property name="toolTip">
    <string>我的请假记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionEmployeeInfo">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/login2.png</normaloff>:/images/login2.png</iconset>
   </property>
   <property name="text">
    <string>员工信息</string>
   </property>
   <property name="toolTip">
    <string>员工信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSign_out">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/考勤.png</normaloff>:/images/考勤.png</iconset>
   </property>
   <property name="text">
    <string>签退</string>
   </property>
   <property name="toolTip">
    <string>签退</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionWorkTime">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/加班.png</normaloff>:/images/加班.png</iconset>
   </property>
   <property name="text">
    <string>工作时长</string>
   </property>
   <property name="toolTip">
    <string>工作时长</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionPerson">
   <property name="icon">
    <iconset resource="src.qrc">
     <normaloff>:/images/客户.png</normaloff>:/images/客户.png</iconset>
   </property>
   <property name="text">
    <string>个人信息</string>
   </property>
   <property name="toolTip">
    <string>个人信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
